examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -466,6 +466,21 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 	}
 
+	_composeShaderCode( code, snippet ) {
+
+		// use regex maybe for security?
+		const versionStrIndex = code.indexOf( '\n' );
+
+		let finalCode = code.substr( 0, versionStrIndex ) + '\n\n';
+
+		finalCode += snippet;
+
+		finalCode += code.substr( versionStrIndex );
+
+		return finalCode;
+
+	}
+
```

Comment:
This diff looks odd. Maybe a merge issue?

Reply:
Good catch. Yes, something happened during the merge.
I will update

Reply:
@Mugen87 when do you think this can be merged?

Reply:
It's going to take some time honestly...
This turned into a very big change and I'd like to have a full understanding of what was done.